### PR TITLE
Android Gyroscope uncalibrated sensor

### PR DESCRIPTION
**Pull request by malverick**
Gyroscope uncalibrated sensor api for android platform.
The uncalibrated gyroscope is similar to the gyroscope, except that no gyro-drift compensation is applied to the rate of rotation. Factory calibration and temperature compensation are still applied to the rate of rotation. The uncalibrated gyroscope is useful for post-processing and melding orientation data.

Tested and working fine.
Device - Xiaomi Mi 4i
Android sdk - 24
Android ndk - r9c
Android api - 19
screenshot_2017-03-15-21-53-49-000_org test plyergyrouncalibrated
screenshot_2017-03-15-21-53-57-627_org test plyergyrouncalibrated

### CODE REVIEW

**Comment by malverick** [comment]
This type of sensor is available only for android platform and after digging into the apis provided by windows and apple, I couldn't find any equivalent api for this type of sensor and also there doesn't exist any way of extracting this type of sensor data from existing sensors. 
It anyone finds a possible way of implementing a equivalent api in any other platform, please post here.
Thank you
References - 
https://msdn.microsoft.com/en-us/library/windows/desktop/dd318969(v=vs.85).aspx
https://developer.apple.com/reference/coremotion

**Comment by malverick** [comment]
Initially the feature was implemented as a separate feature but as discussed, now it is added to the current gyroscope implementation. 
Example of gyroscope and gyroscope uncalibrated is also added.

**Comment by malverick** [comment]
Tested and working fine
Device - Xiaomi Mi 4i
Android sdk - 24
Android ndk - r9c
Android api - 19
index
index1

File: plyer/facades/gyroscope.py [hunk 1]
```diff
@@ -36,13 +49,40 @@ class Gyroscope(object):
...
+        Along x-axis: angular speed around the X axis in rad/s
+        Along y-axis: angular speed around the Y axis in rad/s
+        Along z-axis: angular speed around the Z axis in rad/s
+
+        Returns (None, None, None) if no data is currently available.
+        '''
+        return self.get_rotation()
+
+    @property
+    def rotationUncalib(self):
```
Comment (dessant) [inline plyer/facades/gyroscope.py]: Is camel casing frequent for other methods and properties in plyer facades?

Comment (malverick) [inline plyer/facades/gyroscope.py]: not actually. I will change it and use `underscore`

**Comment by malverick** [comment]
All checks have failed due to `pep8` error in `master` branch.
#322

**Comment by malverick** [comment]
closed in favor of #337

**Pull request closed without merging**